core/executor/src/precompiles/tests.rs

```diff
@@ -203,3 +214,100 @@ fn test_blake2f() {
     let output = hex_decode("ba80a53f981c4d0d6a2797b69f12f6e94c212f14685ac4b74b12bb6fdbffa2d17d87c5392aab792dc252d5de4533cc9518d38aa8dbf1925ab92386edd4009923").unwrap();
     test_precompile!(Blake2F, input, output, 12);
 }
+
+#[test]
+fn test_verify_cmbt_proof() {
+    let build_indices = |input: &[u32]| -> Vec<u32> {
+        let mut set = HashSet::with_capacity(input.len());
+        input.iter().for_each(|i| {
+            set.insert(*i);
+        });
+        set.into_iter().collect()
+    };
+
+    // Tx order is 0, 1, 2
+    let tx_hash_0 = H256::from_low_u64_be(0x00).0.pack();
+    let tx_hash_1 = H256::from_low_u64_be(0x01).0.pack();
+    let tx_hash_2 = H256::from_low_u64_be(0x02).0.pack();
+
+    let witness_hash_0 = H256::from_low_u64_be(0x10).0.pack();
+    let witness_hash_1 = H256::from_low_u64_be(0x11).0.pack();
```

Comment:
what is witness hash and how to get it?

Reply:
Witnesses hashes is calculate as https://github.com/nervosnetwork/ckb/blob/378835b/util/gen-types/src/extension/calc_hash.rs#L265-L271